client/src/app/dialog/dialog.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Inject } from '@angular/core';
+import { Member } from '../_models/index';
+import { MatDialogRef, MAT_DIALOG_DATA, MatDialog } from '@angular/material';
+import { FormControl, Validators, FormGroup } from '@angular/forms';
+
+@Component({
+    selector: 'app-dialog-overview-example-dialog',
+    templateUrl: './dialog.component.html',
+})
+export class DialogComponent {
```

Comment:
Can you change the name to something more meaningful like `CodeDialogComponent` or something like that ?